exalearn-ui/src/app/modules/app-routing/app-routing.module.ts

```diff
@@ -0,0 +1,17 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from "@angular/router";
+import { MainPageComponent } from "../../pages/main-page/main-page.component";
+
+const routes: Routes = [
+  { path: 'main', component: MainPageComponent },
+  { path: '', redirectTo: '/main', pathMatch: 'full' },
+  // { path: 'history', component: MainPageComponent },
```

Comment:
Please, remove your technical comments from code, try to store it in notepad, for instance. We leave only important for development comments, such as a description of the function and its parameters, an explanation of some of the actions, etc.

Reply:
comments deleted